css/properties.json

```diff
@@ -399,7 +399,7 @@
     "groups": [
       "WebKit Extensions"
     ],
-    "initial": "<code>currentcolor</code>",
+    "initial": "<code>currentColor</code>",
```

Comment:
Please keep `currentcolor` lowercase! This makes it consistent with other keywords and the spec.

Sebastian